src/provider/persistence/dataLayer/repositories/HistoryRepository.ts

```diff
@@ -1,12 +1,11 @@
-import {Notification, SenderInfo} from '../../../../client/Notification';
-import {ReturnResult, VoidResult} from '../../../model/Result';
 import {PageInfo} from '../../models/PageInfo';
 import {IDatastore} from '../IDatastore';
+import {StoredNotification} from '../../../model/StoredNotification';
 
-import {Repository} from './Repository';
+import {Repository, ReturnResult, VoidResult} from './Repository';
 
 // Shorthand for the intersection-type stored in the repositry
-type DataType = Notification&SenderInfo;
+type DataType = StoredNotification;
```

Comment:
Worth just getting rid of this?

Reply:
Probably. My philosophy was minimal changes to the persistence stuff, but probably no harm in deleting it